src/app/modals/createwallet/passphrase/passphrase.service.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { TestBed, inject } from '@angular/core/testing';
 
-import { SharedModule } from '../../../shared/shared.module';
-import { RpcModule } from '../../../../core/rpc/rpc.module';
 
-import { PassphraseService } from './passphrase.service';
-import { IpcService } from '../../../../core/ipc/ipc.service';
+import { SharedModule } from '../../../wallet/shared/shared.module';
+import { CoreModule, IpcService} from '../../../core/core.module';
+import { ModalsModule, PassphraseService } from '../../modals.module';
 
 describe('PassphraseService', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [
         SharedModule,
-        RpcModule.forRoot()
+        CoreModule.forRoot(),
+        ModalsModule.forRoot()
```

Comment:
` src/app/modals/createwallet/passphrase/passphrase.service.spec.ts`

Might want to steer clear of importing ModalsModule in the test of a subcomponent of it.